src/features/pass-survey/ui/ActivityItem.tsx

```diff
@@ -383,7 +387,10 @@ function ActivityItem({
         </Box>
       );
       break;
-
+    case 'Unity':
+      <Box flex={1}>
+        <UnityView config={pipelineItem.payload} />
```

Comment:
`UnityView` also has a `< View style={{ flex: 1 }}>` surrounding it, it this necessary?